src/interfaces/store-interfaces/state-model-interface.ts

```diff
@@ -36,10 +36,6 @@ export interface StateModelIF {
     shareStructure: boolean
   }
   validationFlags: ValidationFlagsIF
-  validFlags: {
-    // FUTURE: move all flags here
-    defineCompanyStep: boolean
```

Comment:
This flag was read by SummaryDefineCompany (which I deleted) and never set, so I deleted this too.